# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrtools/devfs.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./cdda2wav/local.cnf.in.orig	Wed Mar 21 14:25:30 2001
+++ ./cdda2wav/local.cnf.in	Wed Mar 21 14:25:56 2001
@@ -43,8 +43,8 @@
 
 ############# Sound device support
 #to disable sound support comment out the corresponding line with HAVE_SOUND
-OSS_SOUND_DEV=	'"/dev/dsp"'
-SUN_SOUND_DEV=	'"/dev/audio"'
+OSS_SOUND_DEV=	'"/dev/sound/dsp"'
+SUN_SOUND_DEV=	'"/dev/sound/audio"'
 
 HAVE_OSS=	@HAVE_OSS@
 HAVE_SUNSOUND=	@HAVE_SUNSOUND@
